index.html

```diff
@@ -339,7 +358,9 @@ <h1>bgp.cheap</h1>
 
                                         <tr>
 
-                                                <td>Suble</td>
+                                                <td><img width="35px" src="https://suble.io/_nuxt/img/suble_transparent.ba53449.png">
```

Comment:
Image will be broken when website gets deployed, should be downloaded like the other images.

Reply:
Agreed

Reply:
Perfect, thank you.